vendor/assets/stylesheets/jquery.ui.spinner_custom.scss

```diff
@@ -0,0 +1,12 @@
+span.ui-spinner {
```

Comment:
this file shouldn't be under vendor, but under app (similar to other scss files we have for overriding defaults from libraries)